src/main/java/de/torui/coflsky/CoflSkyCommand.java

```diff
@@ -20,201 +20,223 @@
+                        if (Config.BaseUrl.contains("localhost")) {
+                            CoflSky.Wrapper.startConnection();
+                            Config.BaseUrl = "https://sky.coflnet.com";
+                        } else {
+                            CoflSky.Wrapper.initializeNewSocket("ws://localhost:8009/modsocket");
+                            Config.BaseUrl = "http://localhost:5005";
+                        }
+                        sender.addChatMessage(new ChatComponentText("toggled dev mode, now using " + Config.BaseUrl));
+                        break;
+                    case "status":
+                        sender.addChatMessage(new ChatComponentText(StatusMessage()));
+                        break;
+                    case "reset":
+                        HandleReset();
+                        break;
+                    case "connect":
+                        if (args.length == 2) {
+                            String destination = args[1];
+
+                            if (!destination.contains("://")) {
+                                destination = new String(Base64.getDecoder().decode(destination));
+                            }
+                            sender.addChatMessage(new ChatComponentText("Stopping connection!"));
+                            CoflSky.Wrapper.stop();
+                            sender.addChatMessage(new ChatComponentText("Opening connection to " + destination));
+                            if (CoflSky.Wrapper.initializeNewSocket(destination)) {
+                                sender.addChatMessage(new ChatComponentText("Success"));
+                            } else {
+                                sender.addChatMessage(new ChatComponentText("Could not open connection, please check the logs"));
+                            }
+                        } else {
+                            sender.addChatMessage(new ChatComponentText("§cPleace specify a server to connect to"));
+                        }
+                        break;
+                    case "openAuctionGUI":
+                        // Example args: (message has multiple parts as it includes spaces)
+                        // ["openAuctionGUI", "/viewauction", "auctionUUID", "messagePart1", "messagePart2", ...]
+
+                        String[] callbackArgs = new String[3];
+                        String[] message = new String[args.length - 3];
+
+                        System.arraycopy(args, 0, callbackArgs, 0, 3);
+                        callbackArgs[0] = "callback";
+                        System.arraycopy(args, 3, message, 0, args.length - 3);
+
+                        WSCommandHandler.flipHandler.lastClickedFlipMessage = String.join(" ", message).replaceAll("\n", "");
+                        CallbackCommand(callbackArgs);
+                        break;
+                    case "set":
+                        if (args.length >= 2 && args[1].equals("gui")) {
```

Comment:
```suggestion
                        if (args.length >= 2 && args[1].equals("ahbuygui")) {
```
This may not be the only gui there is/will be
I also dislike overriding the set command as the option isn't as synced setting